services/nginx/nginx.conf.template

```diff
@@ -35,20 +35,36 @@ http {
     server localhost:{{ ports.fake_oauth }};
   }
 
-  upstream frontends {
+  upstream frontend {
     least_conn;
     {% for p in range(server.processes) -%}
```

Comment:
Doesn't this now need to become `server.dedicated_frontend_processes`?

Reply:
No, so the way it works is that the frontend gets access to all processes, but the API is restricted to a subset only.

Reply:
In principle we could fully isolate them, but I don't think it's necessary.  If a frontend request happens to go to one of the API processes, that's fine.  Just want to make sure some are reserved specifically for frontend too.

Reply:
Basically, API always is second priority to serve up.  Which sounds weird, but I think is the right call for usability.